README.md

```diff
@@ -3,13 +3,13 @@
 How metamask calculates nonces
 
 ```js
-const NonceTracker = require('nonce-tracker')
+const NonceTracker = require("nonce-tracker");
 
-const nonceTracker = new NonceTracker(config)
+const nonceTracker = new NonceTracker(config);
 
-nonceLock = nonceTracker.getNonceLock('0xselectedEthereumAddress')
+nonceLock = nonceTracker.getNonceLock("0xselectedEthereumAddress");
 
-nonce = nonceLock.nextNonce
+nonce = nonceLock.nextNonce;
```

Comment:
Hm, is this file not being linted? The double quotes should be converted to single quotes by prettier.

Reply:
Oops, good catch! I had accidentally created the `.prettierrc.js` file in the `.circleci` directory, so it was being ignored. I've just fixed that most recent commit and fixed all lint errors.